#include "Server.h"


namespace WZRY
{

    void Server::Response(int, LoginRequest)
    {
        return;
    }
    void Server::Response(int, RegisterRequest)
    {
        return;
    }
    void Server::Response(int, MatchRequest)
    {
        return;
    }
    void Server::Response(int, ChatRequest)
    {
        return;
    }
    void Server::Response(int, SkillRequest)
    {
        return;
    }
    void Server::Response(int, CloseRequest)
    {
        return;
    }


}  // namespace WZRY








